okhttp-tls/src/test/java/okhttp3/tls/HeldCertificateTest.java

```diff
@@ -256,10 +256,21 @@
         + "MEECAQAwEwYHKoZIzj0CAQYIKoZIzj0DAQcEJzAlAgEBBCA7ODT0xhGSNn4ESj6J\n"
         + "lu/GJQZoU9lDrCPeUcQ28tzOWw==\n"
         + "-----END PRIVATE KEY-----\n";
+    String bcPkcs8Pem = ""
+        + "-----BEGIN PRIVATE KEY-----\n"
+        + "ME0CAQAwEwYHKoZIzj0CAQYIKoZIzj0DAQcEMzAxAgEBBCA7ODT0xhGSNn4ESj6J\n"
+        + "lu/GJQZoU9lDrCPeUcQ28tzOW6AKBggqhkjOPQMBBw==\n"
+        + "-----END PRIVATE KEY-----\n";
 
     HeldCertificate heldCertificate = HeldCertificate.decode(certificatePem + pkcs8Pem);
     assertThat(heldCertificate.certificatePem()).isEqualTo(certificatePem);
-    assertThat(heldCertificate.privateKeyPkcs8Pem()).isEqualTo(pkcs8Pem);
+
+    // Slightly different encoding
+    if (platform.isBouncyCastle()) {
+      assertThat(heldCertificate.privateKeyPkcs8Pem()).isEqualTo(bcPkcs8Pem);
```

Comment:
https://lapo.it/asn1js/#ME0CAQAwEwYHKoZIzj0CAQYIKoZIzj0DAQcEMzAxAgEBBCA7ODT0xhGSNn4ESj6Jlu_GJQZoU9lDrCPeUcQ28tzOW6AKBggqhkjOPQMBBw